poet_test.go

```diff
@@ -199,10 +202,13 @@ func TestHarness_CrashRecovery(t *testing.T) {
 	req.Equal(roundsID[1], info.ExecutingRoundsIds[0])
 	req.Equal(roundsID[2], info.OpenRoundId)
 	req.NoError(h.TearDown(true))
+
+	// leave some time for the server to shutdown.
+	time.Sleep(100 * time.Millisecond)
```

Comment:
😢 
It should not spawn the server as a separate process in the first place. But this is a bigger refactor I guess...